package-lock.json

```diff
@@ -114,7 +114,7 @@
         "jest-environment-jsdom": "29.6.4",
         "js-yaml": "4.1.0",
         "mammoth": "1.6.0",
-        "marked": "4.3.0",
+        "marked": "9.0.0",
         "marked-terminal": "5.2.0",
```

Comment:
The 'marked' package version has been updated here as well. As with the package.json, please ensure that this version change does not introduce breaking changes or incompatibility issues.